### DISCUSSION TITLE
Permadeath requires Manage Money permission for players to buy a new body

### DISCUSSION THREAD

**Discussion by WJohn** [FakeFishGames/Barotrauma#14176] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When a player dies in Permadeath, there are two methods of revival: Possessing a bot, or buying themselves a new body through the HR manager as though they were alive. However, buying oneself a body requires the Manage Money permission to be allowed to do it. This problem is compounded by the fact that the host must do this for each and every dead player -- and that dead players are unable to talk to the host or remind or poke them with "Hey, can you revive me?"

Ideally, revival through the HR manager should not require any Permissions to be granted to players as it is cumbersome to have to do each time, and easy to forget when the player cannot communicate that they need it.

### Reproduction steps

1. Ensure Permadeath mode is enabled
2. Play through a Multiplayer Campaign with other Users
3. Proceed to an Outpost with sufficient money in the Bank
4. Have a non-admin player die
5. Observe whether the player can respawn through the HR manager without any Permissions

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.8.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Vykn** [comment]
A good way to do this could be to allow a respawning player to select a character that they would like to buy and possess, and it goes to the crew as a vote.

Also it would've been nice if bots could be purchased beforehand and didn't physically exist until they were needed (the reserve bench) as was promised. It's not fun having to babysit bots just so people can respawn quickly. _Especially_ in the endgame, where you can't buy new crew.

Additionally, there's the problem of being unable to change the name of a character. The respawning player is stuck with the randomly generated name, making it a lot more difficult to figure out who's who.

**Comment by JeroonBosch** [comment]
Yep, this is something on our radar and definitely something we will improve on

> Also it would've been nice if bots could be purchased beforehand and didn't physically exist until they were needed (the reserve bench) as was promised.

This is still planned, but unfortunately couldn't make it into the first iteration. We opted to release the permadeath as suitable for playing with friends (where everyone is trusted with money management) rather than delaying the whole mode, but improvements for larger/public servers such as the promised reserve bench will still be added.

> Additionally, there's the problem of being unable to change the name of a character.

This is also something we're looking into. For a more roleplaying run with a smaller crew, I think this can be fun and fitting, but in many other cases (such as large servers) this can be very confusing.

**Comment by Rokvach** [comment]
Fixed and tested, coming in the next hotfix patch. Closing.
